include/epiworld/model-bones.hpp

```diff
@@ -505,7 +507,8 @@ class Model {
         std::string fn_reproductive_number,
         std::string fn_generation_time,
         std::string fn_active_cases,
-        std::string fn_outbreak_size
+        std::string fn_outbreak_size,
+        std::string fn_hospitalizations = ""
```

Comment:
Again, I want consistency:


```suggestion
        std::string fn_hospitalizations
```